test/blackbox-tests/utils/dune_stat.ml

```diff
@@ -0,0 +1,21 @@
+open Stdune
```

Comment:
A docstring in here explaining why this utility was introduced (i.e., because it's portable) might be helpful for future readers.

Reply:
I've added an mli  to explain the purpose of this utility.